extras-requirements.txt

```
@@ -11,14 +11,11 @@
 boto3~=1.9, <1.17.107
 botocore>=1.20.106,<1.20.107
 aiobotocore~=1.4.0
-# aioitertools 0.9 and above requires typing-extensions>4, which conflicts with kfp
-aioitertools<0.9
 s3fs~=2021.8.1
 # https://github.com/Azure/azure-sdk-for-python/issues/24765#issuecomment-1150310498
 msrest~=0.6.21
-# azure-core>=1.23.0 uses ParamSpec, and hence depends on typing_extensions>=4
-azure-core<1.23
-azure-storage-blob~=12.0
+azure-core>=1.23
+azure-storage-blob>=12.13
```

I guess it should be
```suggestion
azure-core~=1.23
azure-storage-blob~=12.13
```